src/mscorlib/shared/System/Threading/Tasks/ConcurrentExclusiveSchedulerPair.cs

```diff
@@ -28,8 +28,8 @@ namespace System.Threading.Tasks
     [DebuggerTypeProxy(typeof(ConcurrentExclusiveSchedulerPair.DebugView))]
     public class ConcurrentExclusiveSchedulerPair
     {
-        /// <summary>A dictionary mapping thread ID to a processing mode to denote what kinds of tasks are currently being processed on this thread.</summary>
-        private readonly ThreadLocal<ProcessingMode> m_threadProcessingMode = new ThreadLocal<ProcessingMode>(() => ProcessingMode.NotCurrentlyProcessing);
+        /// <summary>A processing mode to denote what kinds of tasks are currently being processed on this thread.</summary>
+        private readonly ThreadLocal<ProcessingMode> m_threadProcessingMode = new ThreadLocal<ProcessingMode>();
```

Comment:
I guess this is fine.  I was trying to be explicit about it and not trust that NotCurrentlyProcessing == 0, but, ok.